HW_5-6.md

```diff
@@ -0,0 +1,27 @@
+### Awk /ɔːk/
+#### 1. What is the most frequent browser?
+```
+awk '{print $12}' access.log | sort | uniq -c | sort -rn
+```
+
+#### 2. Show number of requests per month for ip 216.244.66.230 (for example: Sep 2016 - 100500 reqs, Oct 2016 - 0 reqs, Nov 2016 - 2 reqs...)
+```
+grep "216.244.66.230" access.log | cut -d":" -f1 | awk -F"/" '{print "reqs for " $2" "$3}' | uniq -c | sort -nr
```

Comment:
It would be better if you used awk not just for printing. Awk it is also a language, so you can solve this task without using grep and cut.